<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1255</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Up Button</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>591</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Down Button</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="outputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>591</width>
      <height>111</height>
     </rect>
    </property>
    <property name="centerOnScroll">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>85</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_2" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>591</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="TDTconnectButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>340</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>27</height>
     </size>
    </property>
    <property name="text">
     <string>Tap/Double tap on</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tapButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>310</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Waiting..</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doubleButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>310</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Waiting..</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectTDTButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>370</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Tap/Doubletap off</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1255</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
